### PR TITLE
add ag-grid info

### PR DESCRIPTION
**Pull request by sknep**
See https://github.com/RocketCommunicationsInc/astro-components/pull/20

https://deploy-preview-86--astrouxds.netlify.app/patterns/table/#complex-tables

I feel like the content here is a little light, but there isn't much to say about ag-grid other than we think it's cool and we have some CSS for folks who want to use it.

### CODE REVIEW

File: _content/patterns/table.md [hunk 1]
```diff
@@ -39,3 +39,16 @@ Tables use a familiar Selection/Action model. Selection is accomplished by click
 
 ![Table with multiple selection and actions in footer.](/img/components/table-action-footer-cms.png "Table with multiple selection and actions in footer.")
 
+## Complex Tables
+
+For more complex tables, we recommend using either the community or enterprise tier of [ag-grid](https://www.ag-grid.com/). While we cannot provide support for ag-grid or its many features, we do provide light and dark variants of Astro in a theme file consumable by ag-grid.
+
+We maintain a [separate repo for the AG-grid Astro theme](https://github.com/RocketCommunicationsInc/astro-ag-Grid). Please visit the following links to get started:
+
+* Documentation: https://github.com/RocketCommunicationsInc/astro-ag-Grid/#astro-ag-grid-theme
```
Comment (sknep) [inline _content/patterns/table.md:48]: I prefer this to named links but understand if y'all don't!

File: _content/patterns/table.md [hunk 2]
```diff
@@ -39,3 +39,16 @@ Tables use a familiar Selection/Action model. Selection is accomplished by click
 
 ![Table with multiple selection and actions in footer.](/img/components/table-action-footer-cms.png "Table with multiple selection and actions in footer.")
 
+## Complex Tables
+
+For more complex tables, we recommend using either the community or enterprise tier of [ag-grid](https://www.ag-grid.com/). While we cannot provide support for ag-grid or its many features, we do provide light and dark variants of Astro in a theme file consumable by ag-grid.
```
Comment (amalyah) [inline _content/patterns/table.md:44]: - Is "recommend" the right term here?
- ag-grid -> ag-Grid

File: _content/patterns/table.md [hunk 3]
```diff
@@ -39,3 +39,16 @@ Tables use a familiar Selection/Action model. Selection is accomplished by click
...
+
+We maintain a [separate repo for the AG-grid Astro theme](https://github.com/RocketCommunicationsInc/astro-ag-Grid). Please visit the following links to get started:
```
Comment (amalyah) [inline _content/patterns/table.md:46]: AG-grid -> ag-Grid

File: _content/patterns/table.md (hunk 2 above)
Comment (dmcalester) [reply _content/patterns/table.md:44]: I think recommend is reasonable with the consideration we may expand Astro support to other table/grid packages.

Concur on ag-Grid capitalization case.

File: _content/patterns/table.md (hunk 1 above)
Comment (dmcalester) [reply _content/patterns/table.md:48]: I would prefer to see named links.

**Review by dmcalester — CHANGES_REQUESTED** [review]
Minor changes. Concur with Amalya’s note on ag-Grid and would prefer to see named links.

**Review by im-cr — APPROVED** [review]

**Review by dmcalester — APPROVED** [review]

**Comment by sknep** [comment]
Reminder not to merge this PR all the way to master (or really, even put it on the draft branch) until the ag-grid storybook page is pushed to master (it's on next now, in that repo)

**Pull request merged**